### DISCUSSION TITLE
Cannot create a new view using Budibase DB

### DISCUSSION THREAD

**Discussion by FlaminWrap** [Budibase/budibase#10414] [Help] [answered]
Hello,

I cannot create a new view for Budibase Databases. When I try to make one I get the error "Error fetching view data". These apps were updated from a previous version of Budibase and have a considerable amount of data in them. Creating a new app with test data does not have this problem.

Thanks

![image](https://user-images.githubusercontent.com/97764630/234278414-f46460cd-772c-4b19-9d1a-c59f0129d065.png)

**Comment by FlaminWrap** [comment]
On further investigation the error message is.
{"message":"App context required for quota update","status":500}

![image](https://user-images.githubusercontent.com/97764630/234283672-a666a860-b8d1-4151-934c-f1ff5f6944e8.png)

**Comment by melohagan** [comment]
Hey @FlaminWrap 

Could you provide a bit more information so I can look into this one.

1. Are you on cloud or self-hosted?
2. How many rows does the table have that you are trying to create the view for?
3. Which version of Budibase did you first notice this issue?
4. Does the error present when initially creating the view, or is it upon add a calculation or filter?

**Reply by FlaminWrap** [reply]
1. Self hosted
2. I am not sure, maybe 6,000
3. 2.5.2
4. It happens when initially creating the view, filtering the data on the table screen works fine

**Reply by melohagan** [reply]
Hey @FlaminWrap I tried a table with ~50k rows, and creating the view timed out for me, although I didn't see the same error.

I then tried a table with 1000 rows and it was fine. I did notice there's no pagination in the Views so it will try to load all data at once which could definitely be an issue.

Did you find you were able to create views for that table before upgrading?

**Reply by FlaminWrap** [reply]
Yes I was, I was trying to edit a view that I had created. The table also has attachments. I seems to be an app specific thing as I can create views in other apps too.

**Reply by melohagan** [reply]
Would you be able to send an app export to support@budibase.com please? Then I can take a look.

**Comment by FlaminWrap** [comment, marked as answer]
Just a heads up, looks like
* Updating to 2.9.35
* Unpublishing
* Changing the name of the app
* Publishing
* Unpublishing
* Changing the name of the app to what it was before
* Publishing

Solved this problem.

**Reply by melohagan** [reply]
Thank you very much for sharing your solution! I think there's someone else this may help 👍
